src/Exadel.CrazyPrice/Exadel.CrazyPrice.Tests/Common/Models/UserTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Exadel.CrazyPrice.Common.Models;
+using Exadel.CrazyPrice.Common.Models.Option;
+using FluentAssertions;
+using Xunit;
+
+namespace Exadel.CrazyPrice.Tests.Common.Models
+{
+    public class UserTests
+    {
+        [Fact]
+        public void UserTest()

```

Comment:
Can you test with all roles as Theory

Reply:
[Added Theory](https://github.com/yegorl/exadel_discounts_be/pull/24/commits/31752f8927c1d6ae749ac2f2196ce5dd48e85336)